app/code/Transoft/Blog/Controller/Adminhtml/Post/Save.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
I can't see Blog delete action. Did you create it?

Reply:
No, there's no delete action at all.

Reply:
Please create. You have a delete button, but not delete action.

Reply:
created